index.html

```diff
@@ -3484,6 +3484,31 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional media type transformation specifications can be
+defined to provide implementers with further options for producing objects
+that can be mapped to the base media type.
+        </p>
+        <p>
+Specifications are registered with the [[?VC-SPECS]] and
+MUST result in a transformed, concrete serialization
+that is objectively testable via a conformance test suite.
+        </p>
+        <p>
+Transformations from other media types MUST state the directionallity of the 
+transformation from the options:
+        </p>
+<ul>
+  <li>Unidirectional</li>
+  <li>Bidirectional</li>
+</ul>
+      <p>
+Bidirectional transformations MUST result in a lossless round-trip conversion between 
```

Comment:
This is not testable, and the details matter regarding claims of "losslessness".

CBOR-LD conversations destroy field ordering, and can have weird interactions with date formats... It would probably be better to leave the details regarding what "bidirectional mapping" means, to the spec that defines it.

Reply:
agree, without an example of how this could be tested we should remove it